doc/whatsnew/2/2.15/index.rst

```diff
@@ -39,6 +39,10 @@ False positives fixed
 
   Closes #4951
 
+* Don't report ``import-private-name`` for relative imports.
```

Comment:
I think we can backport in 2.14.4, could you move the changelog please ?

Reply:
Sure, done.